client/src/components/History/CurrentHistory/HistoryPanel.vue

```diff
@@ -418,7 +418,7 @@ function updateFilterValue(filterKey: string, newValue: any) {
 }
 
 function getItemKey(item: HistoryItem) {
-    return item.type_id;
+    return itemUniqueKey(item);
```

Comment:
`item.type_id` would be undefined for inaccessible hdas resulting in expanding all the inaccessible hdas synchronously. the `type_id` in our API is defined as optional and used for caching so our UI should not rely on its presence.

Reply:
Yes, this is also one of the fields that is completely redundant, it can be computed in the client if necessary it is just `${item.type}-${item.id}` nothing more. It is there for legacy reasons I guess.